views/options/profile.jade

```diff
@@ -213,6 +213,15 @@ script(id='partials/options.profile.profile.html', type='text/ng-template')
         button.btn.btn-default(ng-click='_editing.profile = true', ng-show='!_editing.profile')= env.t('edit')
         h4=env.t('displayName')
         span(ng-show='profile.profile.name') {{profile.profile.name}}
+        p
+        small
```

Comment:
This will get converted to html as:
``` html
<p></p>
<small>[stuff inside small tag]</small>
```

Are you intending to have that empty `<p>` tag there? Or was the `<small>` tag supposed to be nested inside it?

If you indent `small` and everything that's supposed to be inside it, it'll render as:

``` html
<p>
    <small>[Stuff inside small]</small>
</p>
```